src/DocumentDB.ChangeFeedProcessor.UnitTests/Utils/DocumentExceptionHelpers.cs

```diff
@@ -37,6 +37,7 @@ public static Exception CreateException(string exceptionType, int subStatusCode,
             HttpResponseHeaders httpResponseHeaders = CreateResponseHeaders();
             httpResponseHeaders.TryAddWithoutValidation("x-ms-substatus", subStatusCode.ToString());
             httpResponseHeaders.TryAddWithoutValidation("x-ms-activity-id", "activityId");
+            httpResponseHeaders.TryAddWithoutValidation("x-ms-retry-after-ms", "100");
```

Comment:
I can guess why we need this but still is not exactly sure. I would say we should have retry-after as optional parameter and not use it by default only in interesting tests for DCE.